sqlmesh/core/snapshot/definition.py

```diff
@@ -708,6 +708,14 @@ def is_materialized(self) -> bool:
     def is_paused(self) -> bool:
         return self.unpaused_ts is None
 
+    @property
+    def normalized_effective_from_ts(self) -> t.Optional[int]:
+        return (
+            to_timestamp(self.model.cron_floor(self.effective_from))
+            if self.effective_from
+            else None
```

Comment:
just return 0 here?

Reply:
0 is falsey

Reply:
I know, right. But I felt like 0 is a bad default for a timestamp value

Reply:
who cares, it's millis

Reply:
eg. why not -1 then?

Reply:
-1 is not falsey

Reply:
I know, I just meant that it's a perfectly reasonable candidate for a default value. I'd rather be as specific as possible in public interface.